Core.Tests/TransactionTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FluentAssertions;
+using System;
+using Xunit;
+
+namespace Core.Tests
+{
+	public class TransactionTests
+	{
+		[Fact]
+		public void Constructor_TransactionIsLoaded_HashIsSet()
+		{
+			var transaction = new Transaction("1111", new Date(2000, 12, 31), new Money(1.99, Currency.UAH), "food", "Groceries");
```

Comment:
Codacy found an issue: [Define a constant instead of using this literal 'Groceries' 12 times.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=6066087)